feconf.py

```diff
@@ -775,6 +786,9 @@ def get_empty_ratings():
 BLOG_DASHBOARD_URL = '/blog-dashboard'
 BLOG_EDITOR_DATA_URL_PREFIX = '/blogeditorhandler/data'
 BULK_EMAIL_WEBHOOK_ENDPOINT = '/bulk_email_webhook_endpoint'
+BLOG_HOMEPAGE_DATA_URL = '/blogdatahandler/data'
+BLOG_HOMEPAGE_URL = '/blog'
+BLOG_AUTHORS_PAGE_URL_PREFIX = '/blog/authors'
```

Comment:
This does not seem to have been changed as you described it in your review comment. It seems misleading because the constant implies multiple authors.

Reply:
Changed.